modules/admin_manual/pages/configuration/server/background_jobs_configuration.adoc

```diff
@@ -9,123 +9,14 @@ performance. For that reason, as a system administrator, you can
 configure background jobs (for example, database clean-ups) to be
 executed without any user interaction.
 
-These jobs are typically referred to as
-https://en.wikipedia.org/wiki/Cron[Cron Jobs]. Cron jobs are commands or
-shell-based scripts that are scheduled to periodically run at fixed
-times, dates, or intervals. `cron.php` is an ownCloud internal process
-that runs such background jobs on demand.
+These jobs are typically referred to as https://en.wikipedia.org/wiki/Cron[Cron Jobs]. 
+Cron jobs are commands or shell-based scripts that are scheduled to periodically run at fixed times, dates, or intervals. 
 
-ownCloud plug-in applications can register actions with `cron.php`
-automatically to take care of typical housekeeping operations. These
-actions can include garbage collecting of temporary files or checking
-for newly updated files using `filescan()` on externally mounted file
-systems.
+To run Cron jobs with ownCloud, you can use the `system:cron.php`.
 
-You can decide how often jobs get processed, we recommend an interval of
-one minute.
+include::{partialsdir}configuration/server/system_command.adoc[]
 
-[[cron-jobs]]
-== Cron Jobs
-
-You can schedule Cron jobs in three ways: xref:cron[Cron], xref:webcron[Webcron],
-or xref:ajax[AJAX]. These can all be configured in the admin settings menu.
-However, the recommended method is to use Cron. The following sections
-describe the differences between each method.
-
-There are a number of things to keep in mind when choosing an automation option:
-
-Firstly, while the default method is AJAX, though the preferred way is
-to use Cron. The reason for this distinction is that AJAX is easier to
-get up and running. As a result, it makes sense (often times) to accept
-it in the interests of expediency.
-
-However, doing so is known to cause issues, such as backlogs and
-potentially not running every job on a heavily-loaded system. What’s
-more, an increasing amount of ownCloud automation has been migrated from
-AJAX to Cron in recent versions. For this reason, we encourage you to
-not use it for too long — especially if your site is rapidly growing.
-
-Secondly, while Webcron is better than AJAX, it too has limitations. For
-example, running Webcron will only remove a single item from the job
-queue, not all of them. Cron, however, will clear the entire queue.
-
-NOTE: It’s for this reason that we encourage you to use Cron — if at all possible.
-
-[[cron]]
```

Comment:
The cron (unix) description was helpful. Why did you completely killed it?
It was just about fixing `cron.php` --> `occ system:cron`

Webcron and Ajax is affected if `cron.php` cant be called directly, ok.

There should be a visible warning about the discontinuing plan in a further release as Webcron and Ajax will then not be available anymore.

I propose an additional note in the changelog as it affects a later UI as those items will as consequence not be available anymore... (@pmaier1 )

Reply:
There was a note in this thread, at one point, about removing references to the old way of working in 10.3. A note saying that they've been deprecated would be good, you're right.